aws-iot-thing/aws-iot-thing.json

```diff
@@ -0,0 +1,80 @@
+{
+  "typeName": "AWS::IoT::Thing",
+  "description": "Resource Type definition for AWS::IoT::Thing",
+  "additionalProperties": false,
+  "properties": {
+    "Id": {
+      "type": "string"
+    },
+    "Arn": {
+      "type": "string"
+    },
```

Comment:
Not sure if it matters, but in our public docs rather than "Id" and "Arn" , we have "thingId" and "thingArn" , do we need to make them the same? How does this model get used?

https://docs.aws.amazon.com/iot/latest/apireference/API_CreateThing.html